src/AggregateFunctions/AggregateFunctionUniq.h

```diff
@@ -96,18 +104,33 @@ struct AggregateFunctionUniqHLL12Data<UUID, false>
     Set set;
 
     constexpr static bool is_able_to_parallelize_merge = false;
+    constexpr static bool is_parallelize_merge_prepare_needed = false;
     constexpr static bool is_variadic = false;
 
     static String getName() { return "unique_hll12"; }
 };
 
+template <>
+struct AggregateFunctionUniqHLL12Data<IPv6, false>
```

Comment:
ipv6 change introduce in this commit https://github.com/timeplus-io/proton/commit/7411080d85ffc0a2981b5d5f42e2a52d3a2ffaf6#diff-28d726caaebff128ca648d66098ce3f23cda1e924336a302934a4829c6680608R114

but actually, this `ipv6` specialization is part of another big pr 
https://github.com/ClickHouse/ClickHouse/pull/50551/files#diff-28d726caaebff128ca648d66098ce3f23cda1e924336a302934a4829c6680608R105

I recommend either porting this full pr, or we remove here?

Reply:
will remove it